OPHD/Common.h

```diff
@@ -46,7 +46,11 @@ enum class Direction
 	East,
 	West,
 	North,
-	South
+	South,
+    NorthWest,
+    NorthEast,
+    SouthWest,
+    SouthEast
```

Comment:
Looks like spaces were used to indent here.

Reply:
Ooof, yeah, that's a change in VS2022's latest things, I've been using the spaces over tabs lately in new projects. Will have to update the text editor to keep this consistent.